examples/next/faust-nx/.env.local.sample

```diff
@@ -1,2 +1,4 @@
 # GraphQL Endpoint
 NEXT_PUBLIC_GRAPHQL_ENDPOINT=http://headless.local/graphql
+NEXT_PUBLIC_WORDPRESS_URL=http://headless.local
```

Comment:
We might need to consolidate `NEXT_PUBLIC_WORDPRESS_URL` and `NEXT_PUBLIC_GRAPHQL_ENDPOINT` into one variable

Reply:
Since users have the ability to set their own graphql endpoint via WPGraphQL settings, `NEXT_PUBLIC_GRAPHQL_ENDPOINT` seems to be a hard requirement.

Grabbing the WordPress URL from `NEXT_PUBLIC_GRAPHQL_ENDPOINT` might be challenging as WordPress multisite which allows for subdirectories to be part of the WordPress URL.

Reply:
I've added support for this in my apollo work: https://github.com/wpengine/faustjs/blob/MERL-427-plugin-apollo-client/packages/faust-nx/src/lib/getGraphqlEndpoint.ts.

@josephfusco Feel free to copy if this gets merged before my PR.